### DISCUSSION TITLE
Append Route::resource(modelNamePlural, 'ControllerName') after php artisan make model -c -r

### DISCUSSION THREAD

**Discussion by terremoth** [laravel/framework#33976] [General]
I think most part of the time, developers when generate a model with -c -r (controller and resource), could have a parameter to pass together to append 
```Route::resource(model-name-plural?, 'ControllerName')``` at ```routes/web.php```

and if the parameter ```--api``` was passed together, it could append on ```routes/api.php```

Also the ```-a/--all``` could be modified to add the route too. We have lot's of things being generated, I think routes could be too.

It would make our life faster. What you guys think about?

**Comment by terremoth** [comment]
In the "model-name-plural" we can follow some naming convention/pattern

This 2 guides can help, I think
https://restfulapi.net/resource-naming/
https://apiguide.readthedocs.io/en/latest/build_and_publish/use_RESTful_urls.html

**Comment by wakebit** [comment]
Not all developers store routes in a `routes/web.php` or `routes/api.php`. Your suggestion is usable only for small/pet projects, where routes is not divided on many files. So, is it hard to write yourself one row?

**Reply by terremoth** [reply]
Yeah dude it's hard, lol

Could have a way to pass an arg like ```php artisan make:model Employee -a admin``` where admin is ```routes/admin.php``` 

There is just no excuses, this is indeed a legit question for those who want to scaffold part of the application. If you don't like the idea, just don't use it. Make yourself your customized routes.

**Comment by terremoth** [comment]
@taylorotwell @TBlindaruk @GrahamCampbell @themsaid @driesvints @lucasmichot  what you guys think about?
